core/domain/suggestion_services.py

```diff
@@ -1075,15 +1075,27 @@ def get_reviewable_translation_suggestions_by_offset(
                 sort_key,
                 language_codes))
     elif len(opportunity_summary_exp_ids) > 0:
-        in_review_translation_suggestions, next_offset = (
-            suggestion_models.GeneralSuggestionModel
-            .get_in_review_translation_suggestions_with_exp_ids_by_offset(
-                limit,
-                offset,
-                user_id,
-                sort_key,
-                language_codes,
-                opportunity_summary_exp_ids))
+        in_review_translation_suggestions = []
+        next_offset = 0
+        if (limit is None and len(opportunity_summary_exp_ids) == 1):
+            in_review_translation_suggestions, next_offset = (
+                suggestion_models.GeneralSuggestionModel
+                .get_reviewable_translation_suggestions_for_single_exploration(
+                    offset,
+                    user_id,
+                    sort_key,
+                    language_codes,
+                    opportunity_summary_exp_ids[0]))
```

Comment:
I think we should put this in a new service method altogether, since the API is fundamentally different (accept only one exploration ID, no offset, no limit).

Reply:
Done, PTAL.